apps/src/templates/Notification.jsx

```diff
@@ -19,10 +19,12 @@ const styles = {
   main: {
     borderWidth: 1,
     borderStyle: 'solid',
-    height: 72,
+    minHeight: 72,
     width: styleConstants['content-width'],
     backgroundColor: color.white,
     marginBottom: 20,
+    display: 'flex',
+    flexFlow: 'wrap',
```

Comment:
Use flewFlow: wrap so that our button wraps to another row if we have text contents that can't break.